docs/_sidebar.md

```diff
@@ -52,13 +52,12 @@
   - [Scaling](/wiki/research/scaling/scaling.md)
     - [Core Changes](/wiki/research/scaling/core-changes/core-changes.md)
     - [EIP-4844](/wiki/research/scaling/core-changes/eip-4844.md)
-    - Blobs
     - Sharding
     - Statelessness
     - Purge
-  - MEV
-    - PBS
-    - Censorship
+  - [MEV](/wiki/research/PBS/mev.md)
+    - [MEV-boost](/wiki/research/PBS/mev-boost.md)
+  - [PBS](/wiki/research/PBS/pbs.md)
```

Comment:
Add a direct link to ePBS on the side bar.

- [ePBS](/wiki/research/PBS/ePBS.md)